cpp/ps0001/ps0001_soin_cpp.cpp

```diff
@@ -1,15 +1,19 @@
-#include<iostream>
+#include <iostream>
 #include <conio.h>
 using namespace std;
 int main()
 {
-	int x, y, z, f;
+	// prompting users for 3 numbers to add 1/2
+	unsigned short x, y, z, f;
 	cin >> x;
 	cin >> y;
 	cin >> z;
+
```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add